WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -139,4 +142,8 @@ class ReaderSortingOptionButton: UIControl {
         label.text = sortingOption.localizedDescription
         iconView.image = sortingOption.image
     }
+    
+    public func setLabelBottomCompensation(_ compensation: CGFloat) {
```

Comment:
Added this method to set bottom compensation regarding to those empty spaces at the top of the cells. I was thinking of setting it directly when creating constraints, but this way we can set it programmatically depending on the setup in the app and this button can be centered by default and compensated if needed.
Previously I did the compensation in `containerInsets` constant in `ReaderTableCardCell.swift` but this way it is much cleaner and we see better what is happening and why.